Doc/whatsnew/3.11.rst

```diff
@@ -1162,14 +1162,17 @@ Optimizations
 Faster CPython
 ==============
 
-CPython 3.11 is on average `25% faster <https://github.com/faster-cpython/ideas#published-results>`_
-than CPython 3.10 when measured with the
+CPython 3.11 is an average of
+`25% faster <https://github.com/faster-cpython/ideas#published-results>`_
+than CPython 3.10 as measured with the
 `pyperformance <https://github.com/python/pyperformance>`_ benchmark suite,
-and compiled with GCC on Ubuntu Linux. Depending on your workload, the speedup
-could be up to 10-60% faster.
+when compiled with GCC on Ubuntu Linux.
+Depending on your workload, the overall speedup could likely be 10-60%.
```

Comment:
We don't know their workload.
```suggestion
Depending on your workload, the overall speedup could be 10-60%.
```

Reply:
I actually included the 10-60% as a hedge to imply that the indicated range was not deterministically what the speedup "could" possibly be for a workload, but rather just the likely range of speedups for most workloads. But I guess it doesn't read that way...